courses/fundamentals_of_ada/030_basic_types.rst

```diff
@@ -1352,6 +1334,212 @@ Simple Static Type Derivation
       type Distance is new Measurement
             range 0.0 .. Measurement'Last;
 
+==========
+Subtypes
+==========
+
+----------
+Examples
+----------
+
+.. include:: examples/080_expressions/subtypes.rst
+
+:url:`https://learn.adacore.com/training_examples/fundamentals_of_ada/080_expressions.html#subtypes`
```

Comment:
This link needs to be updated to match the new location (specified above)

Reply:
Do you mean like a PR to learn?